packages/cli/src/commands/pg/upgrade/run.ts

```diff
@@ -0,0 +1,81 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import heredoc from 'tsheredoc'
+import {getAddon} from '../../../lib/pg/fetcher'
+import pgHost from '../../../lib/pg/host'
+import {legacyEssentialPlan, databaseNameFromUrl, essentialNumPlan, formatResponseWithCommands} from '../../../lib/pg/util'
+import {PgDatabase, PgUpgradeError, PgUpgradeResponse} from '../../../lib/pg/types'
+import * as Heroku from '@heroku-cli/schema'
+import confirmCommand from '../../../lib/confirmCommand'
+import {nls} from '../../../nls'
+
+export default class Upgrade extends Command {
+  static topic = 'pg';
+  static description = heredoc(`
+    On Essential-tier databases, this command upgrades the database's Postgres version.
+
+    On Standard-tier and higher leader databases, this command runs a previously scheduled Postgres version upgrade. You must run ${color.cmd('pg:upgrade:prepare')} before this command to schedule a version upgrade.
+
+    On follower databases, this command unfollows the leader database before upgrading the follower's Postgres version.
+    `)
+
+  static flags = {
+    confirm: flags.string({char: 'c'}),
+    version: flags.string({char: 'v', description: 'Postgres version to upgrade to'}),
+    app: flags.app({required: true}),
```

Comment:
when requiring the `app` flag, we also like to also include the `remote` flag. e.g.
```
remote: flags.remote({char: 'r'}),
```